docs/shows/format.md

```diff
@@ -75,6 +75,11 @@ Step 3:
 
 ## Setting step time
 
+!!! warning "Important Syntax change with mpf 0.80"
+
+    In this chapter the used sytax for the time parameter is for versions up to 0.57. With mpf 0.80
```

Comment:
The updated syntax applies to 0.57 as well, it was 0.56 and before that were less strict.

https://missionpinball.org/config/instructions/config_v6/

Reply:
Have changed that should be solved I believe.